### PR TITLE
Fix for 18.22

### LINKED ISSUE #18: 18.22 API changes break mod
Due to changes in the 18.22 API, mods hotkeys can no set consuming to "all"

### PR DESCRIPTION
**Pull request by toonech**
Fixes #18

Changes hotkey to consuming="none"

### CODE REVIEW

File: data.lua [hunk 1]
```diff
@@ -5,7 +5,7 @@ data:extend{
 		type = "custom-input",
 		name = "inventory-cleanup",
 		key_sequence = "SHIFT + C",
-		consuming = "all"
+		consuming = "none"
```
Comment (asiegman) [inline data.lua:8]: not the maintainer, but according to https://forums.factorio.com/viewtopic.php?p=492101#p492101 it should be `game-only` not `none`

```suggestion
		consuming = "game-only"
```

**Comment by ewanm89** [comment]
Well, `none` is valid, but I agree, `game-only` is more like `all` in effect, which is what is in my pull request in #20

File: data.lua (hunk 1 above)
Comment (toonech) [reply data.lua:8]: According to the [discussion](https://mods.factorio.com/mod/even-distribution/discussion/5eaac3df06db07000ef375cf) on the mod page, this setting consuming to game-only does not work

**Comment by 321freddy** [comment]
#20 already fixes this, but still thanks for the PR :)

**Pull request closed without merging**